src/Components/IncidentList/IncidentList.scss

```diff
@@ -14,4 +14,118 @@
     &:hover {
         cursor: pointer;
     }
-}
\ No newline at end of file
+}
+
+.all-incidents {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+
+    .incidents {
+        display: flex;
+        width: 33rem;
+        flex-direction: column;
+        background: #f5f5f5;
+        border-radius: .4rem;
+        margin: 2rem 0.5rem;
+
+        .incident-title {
+            display: flex;
+            border-radius: .4rem .4rem 0em 0em;
+            padding: 1rem;
+            background: #e4e4e4ba;
+            flex-direction: row;
+            box-shadow: 0 0em 0em rgba(0, 0, 0, 0.16), 0 0.1em 0.2em rgba(0, 0, 0, 0.23);
+
+            .incident-status {
+                font-weight: 500;
+            }
+
+            .incident-count {
+                margin-left: auto;
+                font-size: smaller;
+                color: #5f5f5f;
+                font-weight: 100;
+            }
+        }
+
+        .incident-cards {
+            direction: flex;
+
+            // style for no incidents message
+            .no-incidents {
+                text-align: center;
+                font-weight: 100;
+                color: #b5b5b5;
+                margin: 55% 5rem;
+                font-size: .9rem;
+                height: 100%;
+            }
+
+            .incident-card {
+                border-radius: .4rem;
+                margin: 5%;
+                background: white;
+                box-shadow: 0 0em 0em rgba(0, 0, 0, 0.16), 0 0.1em 0.2em rgba(0, 0, 0, 0.23);
```

Comment:
You don't need unit when the value is zero: `0` instead of `0em`. And again, why are we still using `em`???